### ISSUE TITLE
Some simple readme

### ISSUE THREAD

**Issue by dselivanov** [s-u/ulog#1]
Hi Simon. Thanks for great package (as usual).

Would be great to have simple readme which can help less experienced users:
- example on how to use reference `ulogd` implementations (for me was not that obvious)
- mention standard system loggers like `rsyslog`, `syslog-ng`, `nxlog`. For example I found this introduction article quite useful - https://sematext.com/blog/what-is-syslog-daemons-message-formats-and-protocols/
- may be example on how to use `ulog` with `rsyslog`

If you are interested, I can send PR.

**Comment by s-u** [comment]
I agree that it would be useful. If you have a minute for a PR, that would be great.
